src/composedoperator/tracesimplex.jl

```diff
@@ -0,0 +1,144 @@
+import Base.sign
+import Base.getindex
+
+
+"""
+TraceSimplex(simplex,direction::SVector)
+    simplex plus infinitesimal direction d, for each point p on the simplex, the trace is taken in the p-d direction
+"""
+struct TraceSimplex{T,U}
```

Comment:
maybe ShiftedSimplex or TranslatedSimplex?

Reply:
In essence they are shifted over an infinitisimal distance, maybe add this property in the name? for example InfTransSimplex ?